src/main/Main.hs

```diff
@@ -135,6 +137,7 @@ options = FayCompilerOptions
   <*> switch (long "version" <> help "Output version number")
   <*> switch (long "Wall" <> help "Typecheck with -Wall")
   <*> switch (long "show-ghc-calls" <> help "Print commands sent to ghc")
+  <*> switch (long "ts" <> help "Output TypeScript-File")
```

Comment:
With this we are no longer outputting a JS file, correct? If so I'd like to change this to "Output TypeScript instead of JavaScript".

Reply:
That's right.